static/css/components/work.less

```diff
@@ -29,7 +29,7 @@
 // Center (editionAbout)
 div.editionAbout {
```

Comment:
I think using h3.edition-header should be more than enough here.

I would recommend dropping div.editionAbout from the selector. We are trying to move towards BEM and simplify these selectors.

Reply:
@jdlrobson will this work
![image](https://user-images.githubusercontent.com/64412143/104104887-b6e3f500-52d0-11eb-984b-4179fd4a0bdb.png)

Reply:
or you want me to remove the div.editionAbout h3 selector?

Reply:
Done.